### PR TITLE
Add MediaElement Plugins by single package.json w/ git auto-update

### PR DESCRIPTION
**Pull request by ron666**
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/johndyer/mediaelement-plugins
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 3
   - Count of stars: 2
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
@PeterDaveHello Although this lib hasn't enough popularity, its files structure looks good to be added.
Do we help adding this kind of lib? Or just wait until getting the enough popularity?

**Comment by PeterDaveHello** [comment]
@maruilian11 Why dont you take a look at https://github.com/johndyer/mediaelement before asking ?

**Comment by PeterDaveHello** [comment]
@maruilian11 I don't think a good structure can decide to add a library

File: ajax/libs/mediaelement-plugins/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "mediaelement-plugins",
+  "version": "1.0.0",
+  "filename": "dist/speed/speed.min.js",
```
Comment (maruilian11) [inline ajax/libs/mediaelement-plugins/package.json]: please change to use `speed/speed.min.js`, thanks.

File: ajax/libs/mediaelement-plugins/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "mediaelement-plugins",
+  "version": "1.0.0",
```
Comment (maruilian11) [inline ajax/libs/mediaelement-plugins/package.json]: please remove the `version`.
we mention it in https://github.com/cdnjs/cdnjs#adding-a-library-by-a-single-packagejson

File: ajax/libs/mediaelement-plugins/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "mediaelement",
+    "plugins"
+  ],
+  "author": "John Dyer",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/johndyer/mediaelement-plugins.git",
+    "fileMap": [
+      {
+        "basePath": "",
```
Comment (maruilian11) [inline ajax/libs/mediaelement-plugins/package.json]: please use `dist` as basepath, thanks!

File: ajax/libs/mediaelement-plugins/package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        "files": [
+          "dist/*"
```
Comment (maruilian11) [inline ajax/libs/mediaelement-plugins/package.json]: we prefer use `**/*` in `dist` folder.

**Comment by ron666** [comment]
You can squash them if you want, please. If you can modify anything that you see that needs to be done, I'd appreciate; otherwise, let me know

**Comment by PeterDaveHello** [comment]
I prefer to use that npm or bower git repo, which has less overhead of auto-update process for us, and both of them are for release purpose, makes more sense.

**Comment by PeterDaveHello** [comment]
@ron666 no problem and no worry, we'll be happy to help you solve the issues and you can learn something from it.

For squash process, will need to clone the repository locally which is more hard and require higher hardware resource, so for this part, we can do it for you if you are not interested in trying this, the other parts, I prefer that we give you advice and tips, and let you fix them yourself.

**Comment by ron666** [comment]
@PeterDaveHello OK I'll fix all the issues and then you can squash it, is that ok?

**Comment by PeterDaveHello** [comment]
@ron666 Yes, of course.

**Comment by ron666** [comment]
Ok on it

**Comment by ron666** [comment]
Ok changes in place

**Comment by PeterDaveHello** [comment]
Okay, I just squashed and rebased.

**Comment by ron666** [comment]
@PeterDaveHello Thanks for the help

**Comment by PeterDaveHello** [comment]
@ron666 You're welcome.

**Comment by ron666** [comment]
@maruilian11 All the changes were done

**Comment by ron666** [comment]
@PeterDaveHello `Auto-update setup` do I need to check the box, or is this something I need to put on the repo?

**Comment by PeterDaveHello** [comment]
@ron666 already here: https://github.com/cdnjs/cdnjs/pull/10485/files#diff-7a00ad5ac643c5b91172330b02f85851R14, so no worries

**Comment by ron666** [comment]
I see now. The question is: why is it not showing the next version when you search it?

**Comment by ron666** [comment]
Or it's an error on my side?

**Comment by PeterDaveHello** [comment]
I saw v1.2.0, isn't it the latest? Or what's your result?

**Comment by ron666** [comment]
This is what I'm seeing
screen shot 2017-02-22 at 9 24 33 am

**Comment by PeterDaveHello** [comment]
@ron666 try a force refresh?

**Comment by PeterDaveHello** [comment]
You can click the commits/history button to take a look the update

**Comment by ron666** [comment]
@PeterDaveHello Thanks that did the trick! My apologies for this inconvenience

**Comment by PeterDaveHello** [comment]
@ron666 no worries

**Pull request merged**